app/lib/utils/btcpayserver.js

```diff
@@ -0,0 +1,19 @@
+import get from 'lodash.get'
+
+const parseConnectionString = value => {
+  let config = {}
+  try {
+    config = JSON.parse(value)
+  } catch (e) {
+    return new Error('Invalid connection string')
+  }
+  const configs = get(config, 'configurations', [])
+  const params = configs.find(c => c.type === 'grpc' && c.cryptoCode === 'BTC') || {}
```

Comment:
May be a good idea to extract 'BTC' into a function argument (e.g `coin` or `network`) so it's more pronounced. And make it default to 'BTC'

Reply:
Good call. Have updated